cmake/Reprojucer.cmake

```diff
@@ -2716,8 +2716,8 @@ function(__set_JucePlugin_Build_defines target target_type)
     list(GET plugin_types ${index} plugin_type)
     list(GET define_suffixes ${index} define_suffix)
 
-    if((JUCER_BUILD_${setting_suffix} AND target_type STREQUAL "SharedCodeTarget")
-        OR target_type STREQUAL "${plugin_type}PlugIn")
+    if(target_type STREQUAL "${plugin_type}PlugIn"
+        OR (JUCER_BUILD_${setting_suffix} AND target_type STREQUAL "SharedCodeTarget"))
```

Comment:
I would suggest also changing the order in the 2nd expression, so that it reads `target_type STREQUAL "SharedCodeTarget" AND ...`. This makes both operands to the `OR` start with a comparison on  `target_type`, making it nicely symmetric.

Reply:
Really good idea! Done in 1da7b06f93a06ba4070a6c7c9777597ee5f48e77.